recipes/kit4dl/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "kit4dl" %}
+{% set version = "2023.08b0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  path: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kit4dl-{{ version }}.tar.gz
+  sha256: 7ab8579afdf042b32559b9cc8eac72ed34d3dbd1fba87e0dfa09353cf7d65ce4
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - kit4dl = kit4dl.cli.app:run
+
+requirements:
+  host:
+    - python >=3.10
+    - pip    
+  run:
+    - python >=3.10
+    - pytorch
+    - lightning
+    - torchmetrics
+    - numpy >=1.23.0
+    - pydantic >=1.10.0
+    - scikit-learn >=1.2.0
+    - colormath
+    - tqdm
+    - colorama >=0.4.3,<0.5.0
+    - shellingham >=1.3.0,<2.0.0
+    - rich >=10.11.0,<14.0.0
```

Comment:
These are not listed in the metadata.

```suggestion
```

Reply:
I suggested adding these because in the metadata it's `typer[all]`, which implies:

```
https://github.com/tiangolo/typer/blob/master/pyproject.toml#L67-L70
```

Reply:
```
all = [
    "colorama >=0.4.3,<0.5.0",
    "shellingham >=1.3.0,<2.0.0",
    "rich >=10.11.0,<14.0.0",
]
```